packages/components/src/mobile/inserter-button/index.native.js

```diff
@@ -62,7 +62,8 @@ class MenuItem extends Component {
 				style={ styles.touchableArea }
 				underlayColor="transparent"
 				activeOpacity={ 0.5 }
-				accessibilityLabel={ blockTitle }
+				accessibilityRole="button"
+				accessibilityLabel={ `${ item.title } block` }
```

Comment:
This change might be breaking the e2e tests

Reply:
Indeed. Thanks @ceyhun